src/DefaultTestReporter.js

```diff
@@ -132,13 +132,24 @@ function (config, aggregatedResults) {
     colors.GREEN + colors.BOLD
   );
 
-  var pluralTestSuites =
-      aggregatedResults.numTotalTestSuites === 1 ?
-      'test suite' : 'test suites';
+  if (aggregatedResults.numFailedToRunTestSuites) {
```

Comment:
I don't really like this name. Can we call it something different, like `numFailedTestSuitesWithRuntimeError` or `numRuntimeErrorTestSuites`?

all then names here kinda suck, but let's make the best out of it until I get some time to actually refactor this :)